scripts/github-actions/linux-build_and_test.sh

```diff
@@ -31,36 +31,29 @@ if [[ "$DO_BUILD" == "yes" ]] ; then
     echo "~~~~~~ FIND NUMPROCS ~~~~~~~~"
     NUMPROCS=`python3 -c "import os; print(f'{os.cpu_count()}')"`
     NUM_BUILD_PROCS=`python3 -c "import os; print(f'{max(2, os.cpu_count() * 8 // 10)}')"`
+
     echo "~~~~~~ RUNNING CMAKE ~~~~~~~~"
-    or_die python3 ./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -bt ${BUILD_TYPE} -DENABLE_GTEST_DEATH_TESTS=ON ${CMAKE_EXTRA_FLAGS}
-    or_die cd build-$HOST_CONFIG-${BUILD_TYPE,,}
+    or_die python3 ./config-build.py -bp builddir -hc ./host-configs/docker/${HOST_CONFIG} -bt ${BUILD_TYPE} -DENABLE_GTEST_DEATH_TESTS=ON ${CMAKE_EXTRA_FLAGS}
+    or_die cd builddir
+
     echo "~~~~~~ BUILDING ~~~~~~~~"
     if [[ ${CMAKE_EXTRA_FLAGS} == *COVERAGE* ]] ; then
         or_die make -j $NUM_BUILD_PROCS
     else
         or_die make -j $NUM_BUILD_PROCS VERBOSE=1
     fi
-    if [[ "${DO_TEST}" == "yes" ]] ; then
-        echo "~~~~~~ RUNNING TESTS ~~~~~~~~"
-        make CTEST_OUTPUT_ON_FAILURE=1 test ARGS='-T Test -VV -j$NUM_BUILD_PROCS'
-    fi
+
+    echo "~~~~~~ RUNNING TESTS ~~~~~~~~"
+    make CTEST_OUTPUT_ON_FAILURE=1 test ARGS='-T Test -VV -j$NUM_BUILD_PROCS'
+
     if [[ "${DO_BENCHMARKS}" == "yes" ]] ; then
         echo "~~~~~~ RUNNING BENCHMARKS ~~~~~~~~"
         make CTEST_OUTPUT_ON_FAILURE=1 run_benchmarks
     fi
+
     if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
         echo "~~~~~~ RUNNING MEMCHECK ~~~~~~~~"
         or_die ctest -T memcheck
     fi
 fi
 
-# Note: Azure pipelines requires read/write access for everyone between steps
-find ./axom -type d -exec chmod 755 {} \;
-find ./axom -type f -exec chmod 644 {} \;
```

Comment:
Does GitHub actions not require RW access? I remember this fix being tricky to figure out at the time.

Reply:
Github Actions has different behaviors.. I tried removing this as a whim and it still passed.. Another difference is it clones the repo into a weird directory and dumps you into it. Unlike Azure which clones it to your home directory then leaves you in the home directory.